src/ProgressMeter.jl

```diff
@@ -252,12 +281,20 @@ function printover(io::IO, s::AbstractString, color::Symbol = :color_normal)
     end
 end
 
-function barstring(barlen, percentage_complete; solidglyph="█", emptyglyph=" ")
+function barstring(barlen, percentage_complete; barglyphs::BarGlyphs=BarGlyphs('|','█','█',' ','|'))
```

Comment:
Minor style nitpick: here you use the `BarGlyphs(chars...)` constructor, while above (line 78) you use the `BarGlyphs(string)` constructor. Is there a reason not to use the same one?